### DISCUSSION TITLE
Chat ask your video library

### DISCUSSION THREAD

**Discussion by bbfred** [microsoft/AI-Chat-App-Hack#51] [Q&A] [answered]
Hi all,

Does anyone know where I can find the git for this version of the chat application were you can chat with video?

https://m.youtube.com/watch?v=FyPjrYKErLc

Thanks

Fred

**Comment by pamelafox** [comment]
Hm, is it the Question answering demo linked from this post? https://techcommunity.microsoft.com/t5/ai-azure-ai-services-blog/your-video-insights-promptly-extracted-azure-ai-video-indexer-s/ba-p/3923236

**Reply by bbfred** [reply, marked as answer]
Thanks a lot. I'll look into it!
